specs/phase1/beacon-chain.md

```diff
@@ -520,11 +522,24 @@ def compute_committee_source_epoch(epoch: Epoch, period: uint64) -> Epoch:
 
 ### Beacon state accessors
 
+#### New `get_committee_count_per_slot`
+
+```python
+def get_committee_count_per_slot(state: BeaconState, epoch: Epoch) -> uint64:
```

Comment:
Note to myself: need extra care when calling `get_committee_assignment` at phase 1 fork boundary.